draft-irtf-cfrg-vdaf.md

```diff
@@ -1029,7 +1029,7 @@ produces a public share:
   the nonce and produces a public share, distributed to each of Aggregators,
   and the corresponding sequence of input shares, one for each Aggregator.
   Depending on the VDAF, the input shares may encode additional information
-  used to verify the recovered output shares (e.g., the "proof shares" in Prio3
+  used to verify the recovered output shares (e.g., the "proofs shares" in Prio3
```

Comment:
"proofs shares" is grammatically odd. Even though we agreed this is the right term to use in the code, in English we should say this differently. I'd also suggesting hinting here that usually there will be one proof, but there could be more. (See suggestion.)
```suggestion
  used to verify the recovered output shares (e.g., the "shares of the of the proof(s)" in Prio3
```